buildpack.md

```diff
@@ -760,9 +744,22 @@ The purpose of launch is to modify the running app environment using app-provide
 
 ### Provided by the Lifecycle
 
+#### Targets
+
+The following environment variables MUST be set by the lifecycle during `detect` and `build` phases to describe the target runtime image.
```

Comment:
here we say "MUST" but then below most of the env vars say "(optional)"  - just a little dissonance for me. 

is it reasonable to have two blocks, like, "the following vars MUST" ... and then "the following vars MAY" 

or combine the statements like, "the following vars MUST, or MAY, respectively, be set..."

Reply:
How about something like

> The following environment variables MUST be set by the lifecycle during `detect` and `build` phases to describe the target runtime image, if inputs are provided.

Reply:
Updated in c985eeb